tracer/build/artifacts/dd-dotnet.cmd

```diff
@@ -0,0 +1,2 @@
+@echo off
+"%~dp0win-x64\dd-dotnet.exe" %*
```

Comment:
Just to confirm, in the x86 MSI, we're still putting it in the `win-x64` folder?

Reply:
Yup